prime-dotnet-webapi/ViewModels/Enrollee/_EnrolleeMappingProfile.cs

```diff
@@ -23,10 +24,14 @@ public EnrolleeMappingProfile()
                     !src.Submissions.OrderByDescending(s => s.CreatedDate).FirstOrDefault().Confirmed
                     && src.PreviousStatus.StatusCode == (int)StatusType.RequiresToa
                 ))
+                .ForMember(dest => dest.LinkedEnrolleeId, opt => opt.MapFrom(src => (src.PaperEnrolment == null) ? src.LinkedEnrolment.EnrolleeId : src.PaperEnrolment.PaperEnrolleeId))
+                .ForMember(dest => dest.PossiblePaperEnrolmentMatch, opt => opt.MapFrom(src => unlinkedPaperEnrolments.Any(e => e.DateOfBirth.Date == src.DateOfBirth.Date)))
                 .ForMember(dest => dest.Confirmed, opt => opt.MapFrom(src => src.Submissions.OrderByDescending(s => s.CreatedDate).FirstOrDefault().Confirmed == true));
 
             CreateMap<Enrollee, EnrolleeDTO>()
                 .ForMember(dest => dest.Confirmed, opt => opt.MapFrom(src => src.Submissions.OrderByDescending(s => s.CreatedDate).FirstOrDefault().Confirmed == true))
+                .ForMember(dest => dest.LinkedEnrolleeId, opt => opt.MapFrom(src => (src.PaperEnrolment == null) ? src.LinkedEnrolment.EnrolleeId : src.PaperEnrolment.PaperEnrolleeId))
+                .ForMember(dest => dest.PossiblePaperEnrolmentMatch, opt => opt.MapFrom(src => (src.GPID != null && src.GPID.Contains(Constants.PaperGpidPrefix)) ? false : unlinkedPaperEnrolments.Any(e => e.DateOfBirth.Date == src.DateOfBirth.Date)))
                 .ForMember(dest => dest.HasNewestAgreement, opt => opt.MapFrom(src => newestAgreementIds.Any(id => id == src.CurrentAgreementId)));
 
             CreateMap<EnrolleeDTO, EnrolleeViewModel>();
```

Comment:
This is not an issue in the PR and wasn't mentioned in the ticket at the time of creation.
![image](https://user-images.githubusercontent.com/34261366/143925997-699d66e3-7872-4fd1-877c-8120d1f8cd30.png)

Our warning is telling the admin to search, but I have a ticket that will display the potential matches on the right side, under Adjudication >> Status Reasons.

Not sure if changing it to "Use the Enrollee search to determine which paper enrolments match, or see the related paper enrolment under the Adjudication card" would be good, probably will need client feed back

@mtpultz, any thoughts?

Reply:
I would change the wording but keep the alert.  Don't use the word "card" though that's dev speak and means nothing to the average user.  Might be better to leave it as "... see the possible matches linked under Adjudication" then let the client choose what they want displayed.

Reply:
or even "... see the possible matches linked under the Adjudication section"